homework/src/index.js

```diff
@@ -30,17 +34,104 @@
     return elem;
   }
 
-  function main(url) {
-    fetchJSON(url, (err, data) => {
+  function addRow(label, value, tbody) {
+    const tr = createAndAppend('tr', tbody);
+    createAndAppend('td', tr, { class: 'label', text: label });
+    createAndAppend('td', tr, { text: value });
+  }
+
+
+  function renderRepository(info, container) {
+    container.innerHTML = '';
+    const table = createAndAppend('table', container);
+    const tbody = createAndAppend('tbody', table);
+
+    const tr1 = createAndAppend('tr', tbody);
+    createAndAppend('td', tr1, {
+      text: 'Repository: ',
+    });
+    const repository = createAndAppend('td', tr1);
+    createAndAppend('a', repository, {
+      href: info.html_url,
+      target: '_blank',
+      text: info.name
+    });
+
+    const descriptions = `${info.description} `;
+    addRow('Description: ', descriptions, tbody);
+    const forks = `${info.forks} `;
+    addRow('Forks: ', forks, tbody);
+    const updated = `${info.updated_at} `;
+    addRow('Updated: ', updated, tbody);
+
+  }
+
+  function renderContributors(info, container) {
+    container.innerHTML = '';
+    createAndAppend('h3', container, { text: 'Contributors' });
+    const ul = createAndAppend('ul', container);
+
+    info.forEach(contributor => {
```

Comment:
If you rename the parameter `info` to `contributors`, this line could become more readable:

```js
contributors.forEach(contributor => {
```